config/config.go

```diff
@@ -57,6 +58,7 @@ func Get() *SuperKeyWorkerConfig {
 	options.SetDefault("KafkaGroupID", "sources-superkey-worker")
 	options.SetDefault("KafkaTopics", kafkaTopics)
 	options.SetDefault("LogLevel", "INFO")
+    options.SetDefault("LogHandler", os.Getenv("LOG_HANDLER"))
```

Comment:
So sorry, but I have one more thing. It looks like your editor isn't putting tabs in here, can you run `gofmt -w config/config.go` and commit it? Either that or heres a diff:
```diff
diff --git a/config/config.go b/config/config.go
index 68d0245..c03733b 100644
--- a/config/config.go
+++ b/config/config.go
@@ -58,7 +58,7 @@ func Get() *SuperKeyWorkerConfig {
 	options.SetDefault("KafkaGroupID", "sources-superkey-worker")
 	options.SetDefault("KafkaTopics", kafkaTopics)
 	options.SetDefault("LogLevel", "INFO")
-    options.SetDefault("LogHandler", os.Getenv("LOG_HANDLER"))
+	options.SetDefault("LogHandler", os.Getenv("LOG_HANDLER"))
 
 	options.SetDefault("SourcesHost", os.Getenv("SOURCES_HOST"))
 	options.SetDefault("SourcesScheme", os.Getenv("SOURCES_SCHEME"))
@@ -76,7 +76,7 @@ func Get() *SuperKeyWorkerConfig {
 		KafkaGroupID:       options.GetString("KafkaGroupID"),
 		MetricsPort:        options.GetInt("MetricsPort"),
 		LogLevel:           options.GetString("LogLevel"),
-        LogHandler:         options.GetString("LogHandler"),
+		LogHandler:         options.GetString("LogHandler"),
 		LogGroup:           options.GetString("LogGroup"),
 		AwsRegion:          options.GetString("AwsRegion"),
 		AwsAccessKeyID:     options.GetString("AwsAccessKeyID"),
```

Reply:
I see! yes I am using spaces in my tab key. Good eye! thanks!